XenAdmin/Wizards/NewPolicyWizard/NewPolicySnapshotTypePageSpecific.cs

```diff
@@ -166,6 +166,15 @@ public policy_backup_type BackupType
 
         private void RefreshTab(IVMPolicy policy)
         {
+            /* when a policy does not have any VMs, irrespective of
+             * the snapshot type, enable Quiesce 
+             */
+
+            if (policy.VMs.Count == 0)
+                quiesceCheckBox.Enabled = true;
+            else
+                quiesceCheckBox.Enabled = false;
+
             switch (policy.policy_type)
```

Comment:
The case inside switch handles when the policy's snapshot type is quiesce, then irrespective of whether the policy has VMs or not we should enable the quiesce option.